packages/template-retail-react-app/app/components/_app/index.test.js

```diff
@@ -82,4 +85,68 @@ describe('App', () => {
         expect(hreflangLinks.some((link) => hasGeneralLocale(link))).toBe(true)
         expect(hreflangLinks.some((link) => link.hrefLang === 'x-default')).toBe(true)
     })
+
+    test('App component updates the basket with correct currency and customer email', async () => {
+        // Test basket. _app will be manipulating this basket for this test
+        const basket = {
+            basketId: 'asdfgh',
```

Comment:
Nit: Let's give it an id that is more readable. e.g `basket_id`

Reply:
This has been changed to `basket_id`